### DISCUSSION TITLE
How to change the font size in command mode？

### DISCUSSION THREAD

**Discussion by Faker0715** [JetBrains/ideavim#674] [Q&A] [answered]
<img width="1173" alt="image" src="https://github.com/JetBrains/ideavim/assets/36560441/10c0766a-7590-47e3-a844-4ea412b5b5d9">
that's font too big! how can decrease font?

**Comment by citizenmatt** [comment, marked as answer]
I think it's using the editor font. Check that the editor is not zoomed out, then check the size of the editor font
